modules/apps/translation/translation-test/src/testIntegration/java/com/liferay/translation/info/item/updater/test/InfoFormValuesUpdaterTest.java

```diff
@@ -121,29 +119,96 @@ public void testUpdateArticleFromInfoFormValues() throws Exception {
 				article, infoFormValues);
 
 		Assert.assertEquals(
-			"Este es el titulo", journalArticle.getTitle(LocaleUtil.SPAIN));
+			"Este es el titulo", journalArticle.getTitle(_LOCALE_ES));
+
+		Assert.assertEquals(
+			"Este es el resumen", journalArticle.getDescription(_LOCALE_ES));
+
+		Assert.assertEquals(
+			"<p>Este es el contenido</p>",
+			_assertTranslatedContent(
+				journalArticle.getContent(), "name", _LOCALE_US, _LOCALE_ES));
+	}
+
+	@Test
+	public void testUpdateArticleFromInfoFormValuesTranslateDoNotModifyOthersTranslations()
+		throws Exception {
+
+		Map<Locale, String> contentMap = HashMapBuilder.put(
+			_LOCALE_ES, "Este es el contenido"
+		).put(
+			_LOCALE_US, RandomTestUtil.randomString()
+		).build();
+
+		Map<Locale, String> descriptionMap = HashMapBuilder.put(
+			_LOCALE_ES, "Esta es la descripcion"
+		).put(
+			_LOCALE_US, RandomTestUtil.randomString()
+		).build();
+
+		Map<Locale, String> titleMap = HashMapBuilder.put(
+			_LOCALE_ES, "Este es el titulo"
+		).put(
+			_LOCALE_US, RandomTestUtil.randomString()
+		).build();
+
+		JournalArticle article = JournalTestUtil.addArticle(
+			_group.getGroupId(), 0,
+			PortalUtil.getClassNameId(JournalArticle.class), titleMap,
+			descriptionMap, contentMap, LocaleUtil.getSiteDefault(), false,
+			true, _serviceContext);
+
+		InfoFormValues infoFormValues =
+			_xliffTranslationInfoFormValuesImporter.importXLIFF(
+				_group.getGroupId(),
+				new InfoItemClassPKReference(
+					JournalArticle.class.getName(), 122),
+				TranslationTestUtil.readFileToInputStream(
+					"test-journal-article_122_Jap.xlf"));
```

Comment:
Hi @AliciaGarciaGarcia, can you rename these files to use only underscores (`-`)?

Can we also use the language id (`ja-JP`) instead of `Jap` in the file name? That would be more precise and would set a better pattern if we want to add other languages in the future.